src/scripts/main.js

```diff
@@ -1,3 +1,300 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tbody = document.querySelector('tbody');
+const sortTableASC = function(index, type) {
```

Comment:
Reduce code duplication: The sortTableASC and sortTableDESC functions are almost the same. So just combine it to a one sorting function